lib/compress/zstd_ldm.c

```diff
@@ -133,9 +133,10 @@ static size_t ZSTD_ldm_gear_feed(ldmRollingHashState_t* state,
 }
 
 void ZSTD_ldm_adjustParameters(ldmParams_t* params,
-                        const ZSTD_compressionParameters* cParams)
+                               const ZSTD_CCtx_params* cctxParams)
 {
-    params->windowLog = cParams->windowLog;
+    params->windowLog = cctxParams->cParams.windowLog;
```

Comment:
minor:
you could use the same trick you employed in other parsers,
by defining `const ZSTD_compressionParameters* const cParams = &cctxParams->cParams;` at the top of the function, and then most of the rest of the function would remain identical.